src/backend/storeManagers/gog/games.ts

```diff
@@ -572,12 +695,19 @@ export async function moveInstall(
     return { status: 'error', error }
   }
 
-  changeGameInstallPath(appName, moveResult.installPath)
+  await changeGameInstallPath(appName, moveResult.installPath)
+  if (
+    gameInfo.install.platform === 'windows' &&
+    (isWindows || existsSync(gameConfig.winePrefix))
+  ) {
+    await setup(appName, undefined, false)
```

Comment:
Re-run setup to update configs and registry keys. Old games players and Windows users who use Galaxy will be thankful :p